packages/main/src/modules/inspector.ts

```diff
@@ -13,10 +25,96 @@ export async function start() {
   const port = await getAvailablePort();
   inspectorServer = run('http-server', 'http-server', {
     args: ['--port', port.toString()],
-    cwd: path.join(APP_UNPACKED_PATH, './node_modules/@dcl/inspector/public'),
+    cwd: join(APP_UNPACKED_PATH, './node_modules/@dcl/inspector/public'),
   });
 
   await inspectorServer.waitFor(/available/i, /error/i).catch(error => log.error(error.message));
 
   return port;
 }
+
+export async function openSceneDebugger(path: string): Promise<string> {
+  const alreadyOpen = getWindow(path);
+  if (alreadyOpen) {
+    focusWindow(alreadyOpen);
+    return path;
+  }
+
+  const window = createWindow(path);
```

Comment:
would this be a problem for a very long path?

Reply:
mmmm not sure. I've been working with a *very* long path actually and seemed to work fine.